project/MicroService.scala

```diff
@@ -27,8 +27,8 @@ trait MicroService {
     import scoverage.ScoverageKeys
     Seq(
       // Semicolon-separated list of regexs matching classes to exclude
-      ScoverageKeys.coverageExcludedPackages := "<empty>;model.*;config.*;.*(AuthService|BuildInfo|Routes).*",
-      ScoverageKeys.coverageMinimum := 20,
+      ScoverageKeys.coverageExcludedPackages := "<empty>;.*config.*;.*(AuthService|BuildInfo|Routes|JsErrorOps).*",
+      ScoverageKeys.coverageMinimum := 94,
```

Comment:
Isn't there a level which will be good enough from now onwards? 94% seems a bit arbitrary and high